tests/playwright/src/specs/podman-machine-tests.spec.ts

```diff
@@ -165,14 +166,19 @@ test.describe.serial(`Podman machine switching validation `, () => {
 
   test('Clean up rootless podman machine', async ({ page }) => {
     await deletePodmanMachine(page, ROOTLESS_PODMAN_MACHINE_VISIBLE);
+
+    if (isWindows) {
+      await handlePopupDialog(page, 'Yes');
```

Comment:
We have `handleConfirmationDialog` in utilities package, I think it should serve well for this purpose.

Reply:
done